src/CanvasServiceProvider.php

```diff
@@ -29,6 +29,7 @@ public function boot()
         $this->registerPublishing();
         $this->loadViewsFrom(__DIR__.'/../resources/views', 'canvas');
         $this->loadTranslationsFrom(__DIR__.'/../resources/lang', 'canvas');
+        $this->loadFactoriesFrom(__DIR__.'/Storage/factories');
```

Comment:
Since we're not using the seeders anymore we shouldn't need this.

Reply:
I added the factories to allow the user to test the package easily. Do you believe it is better to remove this?